e2e/pre-e2e.bash

```diff
@@ -1,22 +1,5 @@
 #!/bin/bash
 
-WHALE_MNEMONIC=$1
-EVM_WHALE_MNEMONIC=$2
-SCRIPT_CONTEXT=$3
-
-# account generation & funding is done with this script in dev only for convenience
-# in CI the GH Actions workflow will do those steps instead so that the results from the process are written to stdout
-# this script is ran through node child_process by playwright and only stderr is being logged to the terminal
-if [ "$SCRIPT_CONTEXT" == "dev" ]; then
-  if [ ! -d e2e-account-provider ]; then
-    git clone https://github.com/Kava-Labs/e2e-account-provider
-  fi
-  cd e2e-account-provider && make build && cd ..
-#      Comment out to run locally if accounts.json is manually added
-  rm -f accounts.json
-  ./e2e-account-provider/account-provider --mnemonic="$WHALE_MNEMONIC" --evm_mnemonic="$EVM_WHALE_MNEMONIC" --provider_config="./e2e/provider_config.json" || exit 1
-fi
-
```

Comment:
The only thing we will need to do before running tests is ensure that metamask is installed. We will now use a e2e whale account with its credentials in github secrets (for ci) and .env (for local running)